packages/@webex/plugin-authorization-browser-first-party/src/authorization.js

```
@@ -363,34 +403,40 @@ const Authorization = WebexPlugin.extend({
 
           this.eventEmitter.emit('qRCodeLogin', {
             eventType: 'authorizationFailure',
-            data: res.body
+            data: res.body,
           });
         });
-    }, interval * 1000);
+    };
+
+    const schedulePolling = (interval) =>
+      (this.pollingTimer = setTimeout(polling, interval * 1000));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)